robot/impl/constants_no_cgo.go

```diff
@@ -0,0 +1,4 @@
+//go:build no_cgo
+package robotimpl
```

Comment:
I think this stuff can be removed, but want to check with @abe-winter

Reply:
`no_cgo` is different from `no_media` -- it's handling a build-time issue (compiler fails), not a runtime issue (hang)

Reply:
I think I'm just confused why this would be introduced in this PR as it has very little to do with the construction/building of vision services and just uses already built ones.  Further, it looks like its only choosing to build/not build a singular string?

Reply:
Abe explained to me offline that this is necessary as importing the vision package is impossible otherwise.  Going to resolve this